validator/src/main.rs

```diff
@@ -1574,7 +1576,8 @@ pub fn main() {
         )
         .arg(
             Arg::with_name("no_untrusted_rpc")
-                .long("no-untrusted-rpc")
+                .alias("no-untrusted-rpc")
+                .long("no-nonarchetype-rpc")
```

Comment:
This double negation is rough. How about "--archetype-rpc-only" instead?

Reply:
I think `known` works well here.

Reply:
It's now `--only-known-rpc`